src/components/common/data/DataView.tsx

```diff
@@ -99,7 +99,6 @@ function DataView({
 }: DataViewProps) {
   // Can be removed after DR-2483
   const showPanels = panels.length > 0;
-  // Only used for Direct BQ Query
   const isDatasetFiltered = filterStatement.length > 0;
```

Comment:
I originally added this comment, but I'm not sure why - It's not accurate as far as I can tell.